packages/utilities/src/scroll.ts

```diff
@@ -66,24 +74,12 @@ const _makeElementScrollAllower = () => {
     }
   };
 
-  return (element: HTMLElement | null, events: EventGroup): void => {
-    if (!element) {
-      return;
-    }
-
-    events.on(element, 'touchstart', _saveClientY, { passive: false });
-    events.on(element, 'touchmove', _preventOverscrolling, { passive: false });
+  events.on(element, 'touchstart', _saveClientY, { passive: false });
```

Comment:
Ooh prime spot to use EventListener haha (we can do that later ofc)

Reply:
actually David just helped point out, here is not a react componnet/place we can use hook. this will require some deeper thoughts